Form branch

tasnimulhasan07 opened the pull request:
created contact form to match figma design and spent time fixing alignment issues to match figma design as close as possible to make form form works

tasnimulhasan07 committed:
index

add contact from containers and items

tasnimulhasan07 committed:
css

add properties to container and items

tasnimulhasan07 committed:
image

added contact form background image

tasnimulhasan07 committed:
image

image update

tasnimulhasan07 committed:
create form page

tasnimulhasan07 committed:
linter

fix linter

tasnimulhasan07 committed:
linter

linter

tasnimulhasan07 committed:
linter

linter

tasnimulhasan07 committed:
remove space

remove space

tasnimulhasan07 committed:
space fix

spacefix

tasnimulhasan07 committed:
padding

added padding....

tasnimulhasan07 committed:
added padding

added padding.........

tasnimulhasan07 committed:
added space

readded space.....

tasnimulhasan07 committed:
reverted

reverted

danielufeli commented on index.html:
```diff
@@ -258,17 +258,65 @@ <h3><a> Let's connect </a> </h3>
+             action="https://formspree.io/f/mjvlknzp"
+             method="post"
+             id="form">
+             <input
```
- [ ] Nice working and validating your input fields, kindly add left padding to your input fields to give it space as shown in the Figma template.

![](https://user-images.githubusercontent.com/48371156/165287721-391a1179-e2ed-4a89-8d45-f849c944a2d2.png)

danielufeli commented on index.html:
```diff
@@ -258,17 +258,65 @@ <h3><a> Let's connect </a> </h3>
+               placeholder="Write your message here ..."
+               required>
+             </textarea>
+             <button
```
- [ ] The Get in Touch button looks good, but kind of use the text color and border color as shown in the Figma template.

![btn](https://user-images.githubusercontent.com/48371156/165288271-4d64228d-65d7-4366-a258-999a70b748d2.png)

danielufeli commented on index.html:
```diff
@@ -258,17 +258,65 @@ <h3><a> Let's connect </a> </h3>
-
+          </section>
 
+          <section class="contact__me__section">
```
- [ ] Nice working on your form it looks good and works well, just a little change to make it look like what we have in the Figma template, kindly add more space above the title.
![design](https://user-images.githubusercontent.com/48371156/165289802-29b28ee2-8fc8-4525-bafc-1c8f26a3115b.jpg)

danielufeli reviewed (changes requested):
Hi @tasnimulhasan07,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ![ Alt text](https://media.giphy.com/media/QXDqh31t4oNCxVs6m1/giphy.gif)

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Clean Design ✔️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me with @danielufeli. in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Felix45 committed:
Add padding to input element

tasnimulhasan07 committed:
button colour and padding fix

fixed padding issue inside input and fixed colour and font not matching figma dessign

tasnimulhasan07 committed:
p

p causing linter error, had been removed

tasnimulhasan07 committed:
linter errors

fixed all linter errors

tasnimulhasan07 committed:
linter tweeks

tweeking linter issues

MeqdamAlqudah commented on index.html:
```diff
@@ -258,17 +258,65 @@ <h3><a> Let's connect </a> </h3>
+               I’d love to help with it
+           </p>
+           <form
+             class="contact__me__form"
+             action="https://formspree.io/f/mjvlknzp"
+             method="post"
+             id="form">
```
- [ ]  Please as was  requested from the last reviewer please make sure that your design looks exactly like Figma's design 

| Your design | Figma |
| ------------- | ------- |
| ![image](https://user-images.githubusercontent.com/74744787/165345686-b76f05c1-3907-42ac-ac2e-b624368f9050.png) | ![image](https://user-images.githubusercontent.com/74744787/165345989-4666da42-aff1-4e26-9454-27c1ddd1a540.png) |
| - look at the distance between the title and the background
- look at the black line at the bottom of the form you don't have it in your design 
- look at the border of the button in the design there was no border to this button  | |

MeqdamAlqudah reviewed (changes requested):
Hi @tasnimulhasan07 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![](https://media3.giphy.com/media/2aI0NbmSnKo9R4NgFE/giphy.webp?cid=82a1493btm35b2xlylyza6l8e9at47mdtwyn32a2yj4ot0st&rid=giphy.webp&ct=g)
 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tasnimulhasan07 committed:
update css

fixed background alignment issue

tasnimulhasan07 committed:
added image

added image

thecodechaser commented:
@tasnimulhasan07 Please check my messages on slack

tasnimulhasan07 committed:
update index and styles

added bottom black line for contact form

tasnimulhasan07 committed:
linter

fixed linter error

thecodechaser reviewed (approved):
## STATUS: APPROVED 🟢

## Hi @tasnimulhasan07👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

<img src="https://media3.giphy.com/media/g9582DNuQppxC/giphy-downsized-medium.gif"/>

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
